src/App.tsx

```diff
@@ -76,13 +69,13 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && !loaded && !hasError && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
```

Comment:
Divide this line by a few lines.

Reply:
it is not my decition i chenge only redux